(DOCSP-27153): Flutter collections (#2455)
mongodb/docs-realm: source/sdk/flutter/realm-database.txt

Live Objects
~~~~~~~~~~~~

All Realm objects are **live objects**, which means they
automatically update whenever they're modified. Realm emits a
:ref:`notification event <flutter-react-to-changes>` whenever any property
changes.

You can use live objects to work with object-oriented data natively
without an :wikipedia:`ORM <Object-relational_mapping>` tool. Live
objects are direct proxies to the underlying stored data, which means
that a live object doesn't directly contain data. Instead, a live object
always references the most up-to-date data on disk and :wikipedia:`lazy
loads <Lazy_loading>` property values when you access them from a
:ref:`collection <flutter-collections>`. This means that a realm
can contain many objects but only pay the performance cost for data that
the application is actually using.

Valid write operations on a live object automatically persist to the
realm. If you are using :ref:`Atlas Device Sync <flutter-sync>`,
the updates also propagate to any other synced clients.
You do not need to call an update method, modify the realm, or otherwise "push" updates.

.. _flutter-additional-examples:

Realm Flutter SDK Examples
--------------------------

> - Each of the pages in the Flutter SDK documentation contain example code showing

Author: like the landing page? i don't think that's necessary here.

> - Each of the pages in the Flutter SDK documentation contain example code showing

Reviewer: Up to you but IMO a link here would be more consistent with the other bullets that each have a link. I see this as a sort of "what's next" type section.

> - Each of the pages in the Flutter SDK documentation contain example code showing

Author: i'm going to leave linkless. while it's a "what's next" don't think that requires a link. 

tho upon further though, will rephrase to not say 'each of the pages' b/c that's not technically true since some pages don't have examples.

> - Each of the pages in the Flutter SDK documentation contain example code showing

Author: ```suggestion
- Most pages in the Realm Flutter SDK documentation contain example code showing
```
